### DISCUSSION TITLE
Error while exporting model with export_model.py

### DISCUSSION THREAD

**Discussion by Giounona** [PaddlePaddle/PaddleOCR#12236] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

while running 
python export_model.py
-c
configs/rec/rec_svtrnet.yml
-o
Global.pretrained_model=./rec_svtr_tiny_none_ctc_en_train/best_accuracy
Global.save_inference_dir=./inference/rec_svtr_tiny_stn_en

my environment
paddleocr==2.7.0.2
paddlepaddle==2.5.1  

error:
/PaddleOCR/ppocr/modeling/transforms/stn.py", line 105, in forward
        def forward(self, x):
            x = self.stn_convnet(x)
            ~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
            batch_size, _, h, w = x.shape
            x = paddle.reshape(x, shape=(batch_size, -1))
    File "site-packages/paddle/jit/dy2static/convert_call_func.py", line 222, in convert_call
	if is_builtin(func) or is_unsupported(func):
    File "site-packages/paddle/jit/dy2static/convert_call_func.py", line 122, in is_unsupported
	func_in_dict = func == v

**Comment by SWHL** [comment]
请拉取仓库main分支再次尝试
